hack/create_kptfile.py

```diff
@@ -0,0 +1,360 @@
+"""This is a script created for the updating of the GCP Kpt packages.
+
+The purpose of this is to upgrade to using setters and substitutions
+"""
```

Comment:
Do we just use this for upgrade?
How should users create new substitutions when working with blueprint?

Reply:
This is a onetime script. We can delete it in a subsequent PR. If we need to create additional substitutions we can do so using the kpt CLI.

When doing these types of conversions its just a good practice IMO to check in a script so we have a record in case we need to redo it.